docs/releases/2.7.0.rst

```diff
@@ -0,0 +1,235 @@
+    passed to the ``refresh_stats`` management command.
+  - Added ``--calculate-checks`` parameter to the ``refresh_stats`` command.
+  - Incorporated ``refresh_all_stats`` functionality into ``refresh_stats``.
+  - Using dumpdata and loaddata to move between databases is no longer
+    supported.  If you need to move, please use proper SQL scripts instead.
+  - Added a new ``system`` user to attribute to changes done by the management
+    commands.
+  - Command and action logging.
+
+
+- Internal changes:
+
+  - Switched to a custom user model. This merges the data and functionality
+    available in `auth.User` and `PootleProfile` before, and has allowed to
+    remove the dependency on deprecated third party apps that were bundled in
+    the code.
+  - The multiple Suggestion models have been merged into a single model.
+  - Changed the way units needing to be sync'ed to disk is determined. Units
+    now have a unique revision number within the store they belong to and
+    they'll be synchronized based on the `last_sync_revision` field of the
+    store.
+  - Tests have been resurrected.
+  - Upgraded jQuery to 2.x and applied a bunch of fixes to the Tipsy plugin,
+    avoiding ad-hoc hacks to remove dangling tips.
+  - Translation projects now have a ``creation_time`` field.
+  - Dropped code for several external apps from Pootle codebase. Also upgraded
+    to newer versions of those apps.
+  - The default caching in database has been replaced by local-memory caching.
+    Using memcached is still highly recommended.
+  - Fixed and avoided any inconsistencies in the unit's submitter information.
+
+
+- Sysadmins:
+
+  - The registration and authentication layer is now handled by django-allauth.
+    This gives Pootle implicit support for OpenID, OAuth, OAuth2 and Persona
+    sign-in protocols.
+  - Integrated ElasticSearch-based local TM server into Pootle. Unit
+    submissions update the index instantly.
+  - Upgrade: Direct upgrade is now only possible from 2.6.0
+  - The report target for reporting string errors has been dropped in favor of
+    a report email address. The report is now sent using an automatically
+    pre-filled  contact form. If the project doesn't have a report email then
+    the reports are sent to :setting:`POOTLE_REPORT_STRING_ERRORS_EMAIL`.
+
+
+- Administrators:
+
+  - Only admins can set the fuzzy flag on the unit. Non-admin users are not
+    shown the fuzzy checkbox by default, but in case it's displayed (because
+    the unit was already fuzzy, or some other action turned it fuzzy, such as
+    using MT), they are always forced to clear the flag.
+  - Projects can be disabled from the administration page, allowing to hide
+    them temporarily while retaining statistics.
+  - The ``view`` permission is now project-centric, it can be set server-wide
+    or for projects, but not for individual language-project combinations.
+
```

Comment:
Missing: added the `hide` negative permission to allow restricting access to projects (#3345)